### ISSUE TITLE
Default value in input prompt cannot be taken into account

### ISSUE THREAD

**Issue by oganexon** [c4spar/deno-cliffy#46]
When a default value is passed with an input prompt, I am not able to use the default value
```ts
const name: string = await Input.prompt({
    message: "Module name:",
    default: "foo",
    minLength: 2,
    maxLength: 40,
  });
```
![image](https://user-images.githubusercontent.com/52361520/87526309-fc5d9780-c68a-11ea-828e-8e515cbc5f8c.png)

**Comment by c4spar** [comment]
Hi @oganexon,

thx i will take a look.

**Comment by c4spar** [comment]
Hi @oganexon,

the issue should be fixed in v0.11.1.

**Comment by oganexon** [comment]
Awesome.
Thanks !
